src/components/ha-card.ts

```diff
@@ -27,6 +27,7 @@ export class HaCard extends LitElement {
         display: block;
         transition: all 0.3s ease-out;
         position: relative;
+        overflow: hidden;
```

Comment:
this should not be done in `ha-card`, this should be done on the pages using the `ha-card` that are impacted:
```css
ha-card {
    overflow: hidden;
}
```

Reply:
Should be sorted